homeassistant/components/sensor/lirc.py

```diff
@@ -0,0 +1,108 @@
+"""
+LIRC interface to receive signals from a infrared remote control.
+
+This sensor will momentarily set state to various values as defined
+in the .lintrc file which can be interpreted in home-assistant to
+trigger various actions.
+
+Sending signals to other IR receivers can be accomplished with the
```

Comment:
For a future feature, we could choose to integrate it into this component (but not this pr)